prog-common.go

```diff
@@ -6,6 +6,9 @@ package libbpfgo
 */
 import "C"
 
+// BPFFunc is an enum as defined in https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h
+type BPFFunc uint32
+
```

Comment:
I suppose we need to bring here all the consts (enum) together, providing then to the lib consumer.

https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L5786

https://github.com/aquasecurity/libbpfgo/blob/73f0dea092f125ae0c210623bcaa03449e916086/helpers/argumentParsers.go#L1970

Reply:
Good point, got the enum with some helpers functions from the `helpers` package.